core/forms.py

```diff
@@ -201,6 +249,7 @@ class Meta:
         widgets = {
             "child": ChildRadioSelect,
             "start": DateTimeInput(),
+            "type": PillRadioSelect(),
```

Comment:
Lets implement the same on the dropdown fields in `Feeding`?

Reply:
Done